kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerEngine.kt

```diff
@@ -72,6 +85,21 @@ class JazzerEngine(private val config: KFuzzConfig) : KFuzzEngine {
             statsDir.resolve("${file.nameWithoutExtension}.csv").writeText(csvText)
         }
     }
+
+    private fun logProcessStream(
+        inputStream: InputStream,
+        outputStream: OutputStream,
+        log: (String?) -> Unit,
+    ): Thread = Thread {
+        inputStream.bufferedReader().use { reader ->
+            outputStream.bufferedWriter().use { writer ->
+                var line: String?
+                while (reader.readLine().also { line = it } != null) {
+                    log(line)
+                }
+            }
+        }
+    }.also { it.start() }
```

Comment:
`Thread` can be replaced with `thread(start = true)`. So this line became unnecessary